modules/repository-s3/src/main/java/org/elasticsearch/repositories/s3/S3BlobStore.java

```diff
@@ -69,6 +69,8 @@ class S3BlobStore implements BlobStore {
      */
     static final int MAX_BULK_DELETES = 1000;
 
+    static final int MAX_DELETE_EXCEPTIONS = 10;
```

Comment:
I've picked this randomly more or less. Not sure what it should be. I'm not sure having more than this is useful.

Reply:
:+1: fine by me